platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileWriter.java

```diff
@@ -124,21 +116,15 @@ public DataFileWriter(
         this.dataItemSerializer = dataItemSerializer;
         this.creationInstant = creationTime;
         this.path = createDataFilePath(filePrefix, dataFileDir, index, creationInstant);
-        this.lockFilePath = getLockFilePath(path);
-        if (Files.exists(lockFilePath)) {
-            throw new IOException("Tried to start writing to data file [" + path + "] when lock file already existed");
-        }
```

Comment:
Since lock files are now removed, `DataFileCommon.getLockFilePath()` may be obsolete. Besides that, `DataFileCommon.isFullyWrittenDataFile()` needs to be adjusted to not check lock files any longer.

Reply:
Yes, that's right. I updated `DafaFileCommon` as you suggested.